pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/Azure/ARO-RP/pkg/api/admin"
```

Comment:
changes in this file are to default the cluster version in for creation before validation takes place